src/gossip/service.zig

```diff
@@ -1312,6 +1312,7 @@ pub const GossipService = struct {
                     .output = ArrayList(Packet).init(self.allocator),
                     .allocator = self.allocator,
                     .output_limit = &output_limit,
+                    .seed = 19 + i,
```

Comment:
This is used to calculate a sequence of jitter values. It will calculate a different jitter for each task since it has a different seed for each task. But each time a task pool is created overall, that pool will reuse the same set of jitter numbers. Because of that, you could actually pre-calculate these jitters and hardcode them in a const array instead of a using a random number generator.

Do we really benefit from that much determinism? I would have guessed the idea of this jitter is to make it actually behave somewhat randomly.

If you want determinism for tests, I get that. But it could be parameterized in a different way to make it deterministic for tests while also allowing it to generate a novel value on each iteration when running the full validator. Do you think that would be better or worse?

Reply:
good point - f9987d4